src/js/pages/payment.js

```diff
@@ -0,0 +1,17 @@
+export function payment(){
+    let paymentForm = document.querySelector('#payment-form');
+    paymentForm && paymentForm.addEventListener('submit', () => {
+        let name = document.getElementById("input-name");
+        let email = document.getElementById("input-email");
+        let phone = document.getElementById("input-phone");
+        let address = document.getElementById("input-address");
+        let checkValidate = name.value && email.value && phone.value && address.value && address;
```

Comment:
```suggestion
        let name = document.getElementById("input-name").value;
        let email = document.getElementById("input-email").value;
        let phone = document.getElementById("input-phone").value;
        let address = document.getElementById("input-address").value;
        let customerInformation= {
             name,
             email,
             phone,
             address,
        }
        let isValidate = Object.values(customerInformation).some((value) => value.trim())
```
Thử như này nhé. Vì như trên kia thì nhập 1 khoảng space vẫn cho phép đấy